soap/models.py

```
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+"""
+مدل‌های اپ SOAP
+"""
+
+from typing import Any
```

Ruff (0.12.2)

Module level import not at top of file

(E402)